src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.model;
+
+import java.math.BigDecimal;
+
+public class Fruit {
```

Comment:
we do not need this class
quantity is not an inner Fruit property. quantity is a Storage property